workspace/cmd/api1/api1.go

```diff
@@ -14,21 +15,22 @@ import (
 	emptypb "google.golang.org/protobuf/types/known/emptypb"
 )
 
+var ErrorNeedsImplementation = connect.NewError(connect.CodeUnimplemented, errors.New("needs implementation"))
```

Comment:
Nit: In idiomatic Go, it's customary to use `Err` instead of `Error` for these kinds of variables. We don't export any from connect-go but you can see this in e.g. https://pkg.go.dev/github.com/bufbuild/protocompile/reporter#pkg-variables

Reply:
Indeed. Fixed.